system/sha3/wrapper/package.json

```diff
@@ -6,14 +6,14 @@
   "scripts": {
     "build": "npx polywrap build -s image",
```

Comment:
i think we should add `--codegen` flag since it's not inclusive anymore, or we should update the readme to do `yarn codegen && yarn build` (this applies to both wrappers)

Reply:
Thanks! I fixed this